tracing-brave-http/src/main/java/io/micronaut/tracing/brave/http/BraveTracingServerFilter.java

```diff
@@ -108,8 +109,8 @@ public Publisher<MutableHttpResponse<?>> doFilter(HttpRequest<?> request,
                         serverHandler.handleSend(mapResponse(request, response), span);
                     }
                 })
-                .doOnError(throwable -> span.error(throwable).finish());
-
+                .doOnError(throwable -> span.error(throwable).finish())
+                .contextWrite(ctx -> ReactorPropagation.addPropagatedContext(ctx, propagatedContext));
```

Comment:
I'm thinking of making this by default when triggering the execution. In this case, it would be at the controller method execution. FDYT @graemerocher ?